app/src/organisms/OnDeviceDisplay/RunningProtocol/RunPausedSplash.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react'
+import { useTranslation } from 'react-i18next'
+
+import {
+  Flex,
+  Btn,
+  Icon,
+  JUSTIFY_CENTER,
+  ALIGN_CENTER,
+  SPACING,
+  COLORS,
+  DIRECTION_COLUMN,
+  POSITION_ABSOLUTE,
+  TYPOGRAPHY,
+  BORDERS,
+  OVERFLOW_WRAP_BREAK_WORD,
+} from '@opentrons/components'
+import styled from 'styled-components'
+
+interface RunPausedSplashProps {
+  onClose: () => void
+  errorType?: string
+  protocolName?: string
+}
+
+export function RunPausedSplash({
+  onClose,
+  errorType,
+  protocolName,
+}: RunPausedSplashProps): JSX.Element {
+  const { t } = useTranslation('error_recovery')
+
+  let subText: string | null
+  switch (errorType) {
+    default:
+      subText = protocolName ?? null
+  }
+
+  return (
+    <Btn
+      height="100vh"
+      width="100%"
+      justifyContent={JUSTIFY_CENTER}
+      alignItems={ALIGN_CENTER}
+      position={POSITION_ABSOLUTE}
+      flexDirection={DIRECTION_COLUMN}
+      gridGap={SPACING.spacing40}
+      padding={SPACING.spacing40}
```

Comment:
from Figma

![Screenshot 2024-04-11 at 2 07 12 PM](https://github.com/Opentrons/opentrons/assets/474225/20928988-1bed-4e6f-817e-b1bc59e63095)


```suggestion
      padding={SPACING.spacing120}
```

Reply:
Ah this must have been updated, thanks.